src/azure-cli/HISTORY.rst

```diff
@@ -11,6 +11,10 @@ Release History
 
 * Update ADLS version (0.0.45).
 
+**Appservice**
+
+* functionapp: remove deprecated `az functionapp devops-build` command
```

Comment:
This needs to be annotated as a breaking change.

Reply:
@tjprescott 👍 Fixed